rds-monitoring/templates/prometheusrules.yaml

```diff
@@ -18,7 +18,7 @@ spec:
           for: 10m
           labels:
 {{ include "rds-monitoring.alertLabels" . | indent 12 }}
-            severity: warning
+            severity: info
```

Comment:
:thinking: why? Afaik running out of CPU credits can be quite severe...

Reply:
RDS runs in unlimited mode by default: https://aws.amazon.com/rds/instance-types/

> T4g instances accumulate CPU credits when a workload is operating below baseline threshold. Each earned CPU credit provides the T4g instance the opportunity to burst with the performance of a full CPU core for one minute when needed. Amazon RDS T4g instances are configured for Unlimited mode, which means they can burst beyond the baseline over a 24-hour window for an additional charge.

Reply:
hmmm I see. Ok then